services/payments/src/simcore_service_payments/services/auto_recharge_process_message.py

```diff
@@ -147,6 +147,15 @@ async def _perform_auto_recharge(
     )
     credit_result = parse_obj_as(CreditResultGet, result)
 
+    # NOTE: PC/MD Here in the future, it may be better to have just one RPC call to the
+    # web server where we get all additional data we will need, such as user email/address...
+    result = await rabbitmq_rpc_client.request(
+        WEBSERVER_RPC_NAMESPACE,
+        parse_obj_as(RPCMethodName, "get_product_stripe_info"),
```

Comment:
With this you have created yet another channel back to the webserver
I thought we agreed that `products_prices` would be one of the tables accessible by the payments.
Why not access this data directly from the database? You actually have the code in the `postgres_database` library! Something like

```python
async def get_product_stripe_info(
    app: web.Application, *, product_name: ProductName
) -> ProductStripeInfoGet:
    repo = ProductRepository.create_from_app(app)
    return await repo.get_product_stripe_info(product_name)
```

Reply:
Yes, that was also the first thing I tried, but then I realized there was a bit of inconsistency in using SAConnection in the web server compared to AsyncEngine used in the Payment service. As I delved deeper into the issue, this solution seemed more appropriate to me. As mentioned in the comment, I believe that in the next iteration, we should establish just ONE channel to the web server specifically for this use case. Therefore, on the side of the web server, we will prepare everything that is needed (such as credits, invoice data including stripe IDs, addresses, and emails - essentially those things that the web server owns).